File: app/src/main/java/org/michaelbel/moviemade/rest/model/Crew.java
package org.michaelbel.moviemade.rest.model;

import com.google.gson.annotations.SerializedName;

import java.io.Serializable;

import io.realm.RealmObject;

@SuppressWarnings("all")
public class Crew extends RealmObject implements Serializable {

    @SerializedName("credit_id")
    public String creditId;

    @SerializedName("department")
    public String department;

    @SerializedName("id")
    public int id;

    @SerializedName("job")
    public String job;

    @SerializedName("name")
    public String name;

    @SerializedName("profile_path")
    public String profilePath;
}

File: app/src/main/java/org/michaelbel/moviemade/data/entity/Crew.kt
package org.michaelbel.moviemade.data.entity

import com.google.gson.annotations.SerializedName
import java.io.Serializable

data class Crew(
    @SerializedName("credit_id") val creditId: String,
    @SerializedName("department") val department: String,
    @SerializedName("gender") val gender: Int,
    @SerializedName("id") val id: Int,
    @SerializedName("job") val job: String,
    @SerializedName("name") val name: String,
    @SerializedName("profile_path") val profilePath: String,
    @SerializedName("original_language") val originalLanguage: String,
    @SerializedName("original_title") val originalTitle: String,
    @SerializedName("overview") val overview: String,
    @SerializedName("vote_count") val voteCount: Int,
    @SerializedName("video") val video: Boolean,
    @SerializedName("poster_path") val posterPath: String,
    @SerializedName("backdrop_path") val backdropPath: String,
    @SerializedName("title") val title: String,
    @SerializedName("popularity") val popularity: Float,
    @SerializedName("genre_ids") val genreIds: List<Int>,
    @SerializedName("vote_average") val voteAverage: Float,
    @SerializedName("adult") val adult: Boolean,
    @SerializedName("release_date") val releaseDate: String
) : Serializable